templates/cluster-template-ccm.yaml

```diff
@@ -0,0 +1,266 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ${CLUSTER_NAME}
+  namespace: ${NAMESPACE}
+stringData:
+  NUTANIX_PASSWORD: ${NUTANIX_PASSWORD}
+  NUTANIX_USER: ${NUTANIX_USER}
+---
+apiVersion: bootstrap.cluster.x-k8s.io/v1beta1
+kind: KubeadmConfigTemplate
+metadata:
+  name: ${CLUSTER_NAME}-kcfg-0
+  namespace: ${NAMESPACE}
+spec:
+  template:
+    spec:
+      joinConfiguration:
+        nodeRegistration:
+          kubeletExtraArgs:
+            cloud-provider: external
```

Comment:
since this is the only change, wondering if we can create kustomize based file so that we dont need to duplicate cluster-template.yaml

Reply:
this is already a kustomize and this file is the result